cmd/billing.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/speakeasy-api/huh"
+	sdkGenConfig "github.com/speakeasy-api/sdk-gen-config"
+	"github.com/speakeasy-api/speakeasy/internal/charm"
+	"github.com/speakeasy-api/speakeasy/internal/charm/styles"
+	"github.com/speakeasy-api/speakeasy/internal/log"
+	"github.com/spf13/cobra"
+)
+
+var billingCmd = &cobra.Command{
+	Use:   "billing",
+	Short: "Manage billing related operations",
+	Long:  `Commands for managing billing related operations in Speakeasy.`,
+}
+
+var activateWebhooksCmd = &cobra.Command{
+	Use:   "activate-webhooks",
+	Short: "Activate webhooks in the SDK generation configuration",
+	Long:  `Search for and update the SDK generation configuration file to activate webhooks for the configured language.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Can this be moved to a non-anonymous function? Easier to read

Reply:
sure